packages/components/src/select-control/control.js

```diff
@@ -96,21 +96,6 @@ class Control extends Component {
 			event.stopPropagation();
 		}
 	}
-
-	renderButton() {
```

Comment:
My IDE reported that this function is no longer used. I even checked all child components and I couldn't find usage of this function. Yet, pipelines reported that one E2E test failed (`Enabling cash on delivery enables the payment method`), but I can't see why it failed because AFAIK I haven't touched that part of the plugin. I can return this function and execute E2E tests again and check do they pass.